### ISSUE TITLE
Learning curves

### ISSUE THREAD

**Issue by Zomis** [Zomis/NeuralOne#6]
To detect possible bias and variance, make use of learning curves for the training set and cross-validation set.
